docs/Setup.md

```diff
@@ -14,13 +14,13 @@ Start by making sure the command line with Gradle works first, because **all the
     1. Install JVM 19 or greater on your system.  Our setup has been tested with Java 19.  Although a variety of 
        JVM distributions are available and should work, we have settled on recommending [Adoptium/Temurin](https://adoptium.net), because this is the default distribution used by Gradle toolchains.  For Windows or Linux, be sure that the `JAVA_HOME` environment variable points to the right Java version.  Note: If you switch from a newer to an older JVM version, you may see an error like the following `> com.android.ide.common.signing.KeytoolException: Failed to read key AndroidDebugKey from store "~/.android/debug.keystore": Integrity check failed: java.security.NoSuchAlgorithmException: Algorithm HmacPBESHA256 not available`.  A solution is to delete the debug keystore and allow it to be re-generated.
     1. Android Studio has an embedded JVM, although running Gradle tasks from the command line requires a separate JVM to be installed.  Our Gradle scripts are configured to use toolchains to automatically install the correct JVM version.
-1. Configure Rust (Tested with `1.82.0`)
+1. Configure Rust (Tested with `1.84.1`)
     1. [Install Rust](https://www.rust-lang.org/learn/get-started). You will need at least Rust 1.82.  Older versions of Rust will fail.  If you use system packages, check the provided version.
```

Comment:
Is it still the case that "You will need at least Rust 1.82" and not "You will need at least Rust 1.84.1"?

Reply:
Nope, missed a line